### PR TITLE
Remove Caregivers cross-app import from mock forms

### PR DESCRIPTION
**Pull request by longmd**
## Summary
The 1010 team is looking to move it's applications to the CD pipeline. In order to do so, we cannot have any cross-app imports with our applications. This PR converts an imported component to its own local file to remove the cross-app import.

## Related issue(s)
department-of-veterans-affairs/va.gov-team#76209

## Acceptance criteria

- Form description successfully renders using local component
- Caregivers app contains no cross-app imports

### Quality Assurance & Testing

- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution

### CODE REVIEW

**Review by rmessina1010 — DISMISSED** [review]
LGTM

File: src/applications/simple-forms/mock-simple-forms-patterns/pages/mockTextInputSsn.js [hunk 1]
```diff
@@ -5,16 +5,11 @@ import {
...
 
 /** @type {PageSchema} */
 export default {
   uiSchema: {
     ...titleUI('RJSF'),
-    ssn: {
-      ...ssnUI(),
-      'ui:title': 'Social security number',
-    },
     vaFileNumber: {
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/mock-simple-forms-patterns/pages/mockTextInputSsn.js:13]: could you also remove vaFileNumber and 'view:wcv3Title' here and update titleUI? Typically we keep ssn and vaFileNumber together, so might as well remove both

**Review by rhasselle-oddball — DISMISSED** [review]
Thanks for updating this!  One change request (not required, but preferred)

File: src/applications/simple-forms/mock-simple-forms-patterns/pages/mockTextInputSsn.js (hunk 1 above)
Comment (longmd) [reply src/applications/simple-forms/mock-simple-forms-patterns/pages/mockTextInputSsn.js]: Done!

**Review by rhasselle-oddball — APPROVED** [review]

**Pull request merged**